src/nitypes/scalar/_scalar.py

```diff
@@ -69,7 +69,12 @@ def units(self) -> str:
 
     @property
     def extended_properties(self) -> ExtendedPropertyDictionary:
-        """The extended properties for the scalar."""
+        """The extended properties for the scalar.
+
+        NOTE: Data stored in the extended properties dictionary of a waveform is not encrypted.
```

Comment:
reST has syntax for notes
https://docutils.sourceforge.io/docs/ref/rst/directives.html#note